lombok.config

```diff
@@ -0,0 +1,2 @@
+config.stopBubbling = true
+lombok.addLombokGeneratedAnnotation = true
```

Comment:
Please, add empty line at the end of the file

Reply:
`+`